### ISSUE TITLE
schedule rule start from Date.now() not work 

### ISSUE THREAD

**Issue by NextZeus** [node-schedule/node-schedule#468]
# Code

```
let schedule = require('node-schedule')
console.log(Date.now());

let startTime = new Date(Date.now());
var j = schedule.scheduleJob({ start: startTime, rule: '*/1 * * * *' }, function (fireDate) {
    console.log('Time for tea!', fireDate, Date.now());
});

```

## Output

```
1539681925062
Time for tea! 2018-10-16T09:26:00.000Z 1539681960004
```

first time console Time ... not at Date.now(), it delayed about 35s.

**Comment by santigimeno** [comment]
What output were you expecting?

**Comment by NextZeus** [comment]
> What output were you expecting?

I expected output

1539681925062
Time for tea! 2018-10-16T09:26:00.000Z 1539681925062
Time for tea! 2018-10-16T09:26:00.000Z 1539681985062
Time for tea! 2018-10-16T09:26:00.000Z 1539682045062

**Comment by m-schmoock** [comment]
Yes, it should be possible to `scheduleJob(Date.now(), ...)` or  `scheduleJob("now", ...)` .
Unfortunately, it doesnt work reliable by adding onw millisecond:

`scheduleJob(Date.now()+1, ...)` 

Currently, you have to add more (i.e. 100ms, still unreliable though) or handle direct execution in your code :/
